worker/lease/manager.go

```diff
@@ -207,12 +207,21 @@ func (manager *Manager) retryingClaim(claim claim) {
 
 	if success {
 		claim.respond(nil)
+		// This is pretty subtle - in the success case we send a nil
+		// back on the errors channel. That indicates to the main loop
+		// that some lease state has changed and gives it a chance to
+		// loop back around and determine a new wakeup time based on
+		// new leases.
 	} else if lease.IsTimeout(err) {
 		claim.respond(lease.ErrTimeout)
+		manager.config.Logger.Errorf("[%s] retrying failed in handleClaim", manager.logContext)
```

Comment:
I don't think this is an error - if this is expected (which it is at startup) we are retrying and expect it to come good etc, can we make it a Warning?
Can the message be tweaked to something like "[%s] retrying lease claim timeout" ?

Reply:
Ok, sounds good to me.

Reply:
Although I don't think the message you're suggesting works - this is logged when retrying has failed at this level (although the clients further up the stack will try to claim the lease again).